frontend/catalyst/jax_primitives.py

```diff
@@ -294,7 +294,9 @@ def _python_callback_lowering(
     ip = module.body
     attrs = [fn_ty_attr, callback_id, len(args), len(results_ty)]
     with ir.InsertionPoint(ip):
-        callbackOp = CallbackOp(f"callback_{callback_id}", *attrs)
+        # TODO: Name mangling for callbacks
+        name = callback.__name__
+        callbackOp = CallbackOp(f"callback_{name}_{callback_id}", *attrs)
```

Comment:
I meant, why not doing `f"callback_{callback.__name__}_{callback_id}"` directly

Reply:
Ah. I don't like long lines. I can even imagine moving out the fstring out of that call and into a variable.